numba/np/ufunc/parallel.py

```diff
@@ -422,7 +422,14 @@ def select_from_backends(backends):
                 return lib, backend
 
             t = str(config.THREADING_LAYER).lower()
-            namedbackends = ['tbb', 'omp', 'workqueue']
+            namedbackends = config.THREADING_LAYER_PRIORITY
+            if set(namedbackends) != {'tbb', 'omp', 'workqueue'}:
```

Comment:
Should the length of the set `namedbackends` also be checked, should e.g.
```
NUMBA_THREADING_LAYER_PRIORITY="tbb workqueue omp tbb"
```
be valid?